string data_type
---
string data
